internal/types/types.go

```diff
@@ -188,3 +192,20 @@ func deepCopy(value any) any {
 		panic(fmt.Sprintf("types.deepCopy: unsupported type: %[1]T (%[1]v)", value))
 	}
 }
+
+// String is needed to conveniently inform about type or value
+// mismatch when returning an error. For example,
+// when informing that `int32` is an invalid type, in the
+// error message we would like to get `int` and not `int32`.
+func String(v any) string {
```

Comment:
I guess we're going to need this or something similar when implementing more operators

Reply:
I've implemented this method in #544

Reply:
Let's cut it when #544 is merged